dotcom-rendering/src/web/layouts/LiveLayout.tsx

```diff
@@ -585,8 +589,7 @@ export const LiveLayout = ({ CAPIArticle, NAV, format }: Props) => {
 						</GridItem>
 					</StandFirstGrid>
 				</ElementContainer>
-				{CAPIArticle.config.switches.keyEventsCarousel &&
-				CAPIArticle.keyEvents.length ? (
+				{showKeyEventsCarousel && CAPIArticle.keyEvents.length ? (
```

Comment:
Would this work with the `&&` pattern now (rather than a ternary) for consistency?